src/BooleanFlag.php

```diff
@@ -38,7 +39,7 @@ public function __construct(
      *
      * @return Flag
      */
-    public function createBooleanFlag(string $key, string $description, array $tags = []): Flag
+    public function createBooleanFlag(string $key, string $description, array $tags = [], int $rollout = 100): Flag
```

Comment:
**nitpick (non-blocking):** Perhaps update CreateBooleanFlagCommand to support this rollout percentage.